linux/keyman-config/debian/changelog

```diff
@@ -1,6 +1,64 @@
-keyman-config (10.99.1-1) UNRELEASED; urgency=low
+keyman-config (15.0.6) UNRELEASED; urgency=medium
```

Comment:
Is this picking up version 15.0.6 from the master branch? I would expect this to be the last stable version

Reply:
Done